### ISSUE TITLE
Join the 2020 Translators team

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/almanac.httparchive.org#923]
Translators are a group of fluent multilingual technical writers who make the insights of the Web Almanac accessible to new audiences worldwide by translating the contents of the 20+ chapters. To learn more, check out the [Translators' Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide).

Are you interested in joining the fine folks at in the @HTTPArchive/translators team? Simply comment on this thread and we'll send you an invite.

**Comment by techhtml** [comment]
Korean translation volunteer is here.

**Comment by bazzadp** [comment]
> Korean translation volunteer is here.

Hi @techhtml , have you any interest in translating the 2019 edition to Korean? Would keep you busy between now and when the 2020 edition is ready in November-ish, and give you a head start for that of what is involved. If so then will raise an issue for that to give you guidance and track your progress. If not and can only commit later in the year then that's fine too.

**Comment by borisschapira** [comment]
I'm there for the 2020 French edition (still trying to finish 2019, though).

**Comment by techhtml** [comment]
@bazzadp Sure :) I'll be starting to translate 2019 edition to Korean.

**Comment by bazzadp** [comment]
> @bazzadp Sure :) I'll be starting to translate 2019 edition to Korean.

Excellent. Opened #926 for this.

**Comment by bcinarli** [comment]
I can help for Turkish translations

**Comment by bazzadp** [comment]
> I can help for Turkish translations

Excellent @bcinarli ! Same suggestion to you - do you want to make a start on adding Turkish for 2019? Just copy #926 to a new issue and have a read of the comments in there to get started.

**Comment by Mezianide** [comment]
I volunteer for the French translation

**Comment by bazzadp** [comment]
> I volunteer for the French translation

Cool. Welcome @Mezianide - just sent you an invite to the team. The 2019 Almanac is still being translated to French under issue #539 if you want to get suck in straight away. Would be great to get that finished before 2020 comes along so if you're up to it then grab a chapter and start translating. See the [Translator's Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide) for more info or comment on issues #539 and there's a few fellow French speakers who will be glad to help I'm sure.

**Comment by kasiageb** [comment]
Hi, I can help with Polish translation :)

**Comment by domibajan** [comment]
I can help with Polish translation too ;)

**Comment by bazzadp** [comment]
Welcome onboard @kasiageb and @domibjn . I've sent you invites to the team. Once you accept you can assign yourself issues. I've also created #1043 if you want to get started on 2019 content as it will be a good few months before 2020 content is ready.

**Comment by pureise** [comment]
I'd love to help with polish translation too :)

**Comment by chengxicn** [comment]
I could help for Chinese translation, will start from the 2019 release.

**Comment by bazzadp** [comment]
> I could help for Chinese translation, will start from the 2019 release.

Excellent news @chengxicn , I've opened #1067 to track this and sent you an invite to the team so you can be assigned the issue.

**Comment by AppleTsai** [comment]
Hey, I'd love to put my hands in chinese translation as well. *v*

**Comment by bazzadp** [comment]
Would love to have you @AppleTsai ! Can you and @chengxicn coordinate on #1067 and split up the work appropriately? Tag me on that issue with any questions!

**Comment by HakaCode** [comment]
I would love to translate to Portuguese after starting and finishing the 2019 release, or better where possible.

**Comment by bazzadp** [comment]
I created a [Git Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Git-Guide) in our wiki for those of you new to git (or wanting a refresher!).

Also come join us on [`#web-almanac Slack`](https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA) if not already on there.

**Comment by mschork** [comment]
Is there still translation work required? Is the guide appear in German?
Happy to join this team this year and increase my contribution next year

**Comment by bazzadp** [comment]
> Is there still translation work required? Is the guide appear in German?
> Happy to join this team this year and increase my contribution next year

Hi @mschork - we're always looking for more translations! We've not had anyone volunteer to translate to German yet so I've opened issue #1300 for this and listed some details in there as to what needs to be done, in the recommended order of doing them. We're still a while away from having 2020 content ready to translate but you can get started on 2019 in the meantime (and a lot of those templates are actually needed for 2020 anyway).

Ping me on that issue if you've any questions or queries on this and thanks for volunteering!

**Comment by ksakae1216** [comment]
I want to do Japanese translation. 😄

**Comment by MSakamaki** [comment]
participate in Japanese translation.

**Comment by mschork** [comment]
Hey Barry! Sorry for my late reply. Can we have a quick chat on this? I'm
new to all this and would like to better understand the workflow of it.
(hope you are receiving this via mail?)
Cheers,
markus

**Comment by bazzadp** [comment]
Sure. Can talk here (if general information that would be useful to all translators), can talk on [the German translation issue](https://github.com/HTTPArchive/almanac.httparchive.org/issues/1300) or feel free to reach out to me on the [`#web-almanac`](https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA) slack, or on  [Twitter](https://twitter.com/tunetheweb) if you want a more chat-like conversation.

But basically it's the following steps:

- Fork this repo - loads of info in our [Git Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Git-Guide) if new to Git and I'm always available to help.
- Copy the base files from [`src/templates/en/`](https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/templates/en) to `src/templates/de/` and translate from English to German. Most of these are needed before we can translate any of the chapters and they are basically a list of phrases we use to build the web site. Most of it should be relatively easy to understand even if you aren't an expert in HTML or the Jinja2 templating language we use. The [German translation issue](https://github.com/HTTPArchive/almanac.httparchive.org/issues/1300) lists the files to be translated at the top.
- Open a PR with your translation where I will review for technical accuracy. Ideally we get a German speaker to review the language as my German is "nicht so gut"  😀
- Repeat for the other templates.
- Do the same with the chapters in [`src/content/en`](https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/content/en). These are in markdown so again should be fairly easy to understand if you're not that technical. Submit a PR for each chapter.
- Once we have a chapter or two translated, we launch the language and you get your name in lights in our [contributor's page](https://almanac.httparchive.org/en/2019/contributors?teams=translators) and the gratitude of the German web community!

At the moment we're still writing the 2020 content and a lot of the 2019 base templates are needed for them to run anyway so you can get started on 2019 templates and content and then later in the year, switch to 2020 content when it's ready to be translated.

**Comment by PraveenPal4232** [comment]
Hi @bazzadp, I can help with Hindi translation :)

**Comment by bazzadp** [comment]
> Hi @bazzadp, I can help with Hindi translation :)

That would be cool! Sent you an invite to the GitHub team and will open an issue to track this once you accept that.

**Comment by bazzadp** [comment]
Sorry @PraveenPal4232 missed that you'd accepted that. I opened #1402 and assigned to you. Let's continue the discussion there if still interested.

**Comment by PraveenPal4232** [comment]
No problem @bazzadp , it's ok.
Yes, I got assigned, Thanks. I started working on this

**Comment by lex111** [comment]
Hi everyone, I would like to help translate the content into Russian. Is it still possible?

**Comment by bazzadp** [comment]
> Hi everyone, I would like to help translate the content into Russian. Is it still possible?

Absolutely! Always happy to have more volunteers! We've issue #587 for this but didn't make any progress on it so it was closed. Just sent you an invite and will assign that task to you once you accept that and I can answer any questions you might have in that issue.

**Comment by bazzadp** [comment]
@HTTPArchive/translators the first 2020 chapter ([Capabilities](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/src/content/en/2020/capabilities.md)) has been edited, released, and is ready for Translation. The others are still being written and edited so not ready for Translation yet.

I've updated all your issues to add the 2020 chapters (only one for now) in the first comment, above the 2019 chapters, so you can assign yourself as usual.

Until you get at least one 2020 chapter translated, your home page shows the English featured chapter section as a fall back, so would be good to get one 2020 chapter translated ASAP if any of you have time to work on the Capabilities one.

**Comment by bazzadp** [comment]
The Performance and CMS chapters are now available for translating. Please mark your name agains the chapter in your translation issue if you want to translate one of these.

Markup, SEO and HTTP/2 are also very close to being finished editing.

**Comment by antoniomavaro** [comment]
Here is a volunteer for the Italian translation of 2020 Edition.

**Comment by bazzadp** [comment]
Thanks for volunteering @antoniomavaro ! I've just sent you an invite to the team and we already have an issue for the Italian translation in #585 but unfortunately closed it due to lack of activity. Have a read of that and feel free to re-open that issue and ask any questions there.

**Comment by onderceylan** [comment]
I can join @bcinarli on Turkish translations if needed.

**Comment by hijuliancode** [comment]
Hi, I want help with Spanish translations

**Comment by bazzadp** [comment]
> Hi, I want to help with Spanish translations, 😃

Excellent. That's being tracked in #498 and sent you an invite to the team so you can be assigned issues.

**Comment by Zuckjet** [comment]
I can help with simply chinese translation for the 2020 version, especially about web security、progressive web application and so on since my work projects have covered this topic. Is it possible begin with 2020 version, and how can I start ?

**Comment by bazzadp** [comment]
Just comment on that other issue. Unfortunately the security and PWA chapters have not been edited yet so are subject to change. You can see in #1432 which chapters have been edited and are considered complete and ready for translation.

And absolutely, we want 2020 to be priority now and 2019 to be only done after if time permits. Though feel free to pick 2019 Security and PWA chapters if particularly interested in those, while you wait for 2020 ones to be ready.

**Comment by nucliweb** [comment]
I can help with Spanish translation too 😊

**Comment by noah-vdv** [comment]
Hey, I don't know if anyone started it yet but I'm available to help with Dutch translation

**Comment by bazzadp** [comment]
> Hey, I don't know if anyone started it yet but I'm available to help with Dutch translation

Hey @noah-vdv , no one started that yet. Just opened #1750 to track that and tagged you in it. All instructions in there but feel free to ask any questions on that issue. Bedankt voor je vrijwilligerswerk!

**Comment by moniloria** [comment]
Hello! I would like to help with the Spanish Translation!

**Comment by bazzadp** [comment]
> Hello! I would like to help with the Spanish Translation!

Welcome @moniloria! The Spanish translation is being tracked over at #498

**Comment by z1ming** [comment]
Hi, I want to help with Chinese translations, 😃

**Comment by EwaKuros** [comment]
I'd like to help with Polish translation

**Comment by bazzadp** [comment]
> Hi, I want to help with Chinese translations, 😃

@z1ming we have two translation sin progress for [Simplified Chinese](https://github.com/HTTPArchive/almanac.httparchive.org/issues/1067) and [Traditional Chinese](https://github.com/HTTPArchive/almanac.httparchive.org/issues/1106). Which language would you be interested in contributing too? Both have all the scaffolding in place so ready to translate the content - just accept that invite I've just sent you, put your name beside a chapter in the relevant issue and away you go!

> I'd like to help with Polish translation

@EwaKuros we have [a Polish issue](https://github.com/HTTPArchive/almanac.httparchive.org/issues/1043) but unfortunately haven't made much progress on it yet, so it still needs the base templates translated first before we can do the chapters. More details in that issue but basically same thing - tag your name besides the base templates so we know you are working on them, and start translating them!

**Comment by chefleo** [comment]
I volunteer to translate in Italian the Resource Hints chapter 😃.

**Comment by bazzadp** [comment]
Thanks @chefleo - just sent you a GitHub invite to the team!

The Italian translation was being tracked in #585 but it’s not started yet. The base templates will need to be translated first before chapter translations can be published.

**Comment by tymosh** [comment]
Hi, I'd like to help with Ukrainian traslation

**Comment by hanj0415** [comment]
Hello, I want to help with Korean translation for the Caching chapter in 2020.

**Comment by bazzadp** [comment]
Excellent! I've sent you both invites to the team.

@tymosh - we didn't have an issue for Ukranian, so I opened #1873 to track this. Have a read of that and ask any questions on that issue. The Caching chapter is still waiting to be Edited so not quite ready for Translation anyway as that usually introduces a number of changes.

@hanj0415 we had a Korean issue (#926 ), but no progress on it yet, so it needs the base templates translated before we can do any chapters. Do you wanna get started on those?

**Comment by hanj0415** [comment]
@bazzadp Yes I will.

**Comment by tymosh** [comment]
@bazzadp Thank you! Got it.

**Comment by hi-toshi** [comment]
Hi I'd like to help with Japanese translations.

**Comment by bazzadp** [comment]
> Hi I'd like to help with Japanese translations.

Welcome @hi-toshi ! Luckily we've made good progress on Japanese last year in #586 so it's all ready to get straight into the chapter translations. Accept that invite I've sent you, then edit the first comment in #586 and put your name beside a chapter, and then follow the instructions in that comment to translate it and submit a pull request for review!

**Comment by brunopulis** [comment]
I can help with Portuguese translation 😊

**Comment by bazzadp** [comment]
> I can help with Portuguese translation 😊

Excellent! Head on over to #505 and pick a chapter @brunopulis ! Sent you an invite to the team to allow you to edit issues and put your name besides a chapter to “claim” it.

**Comment by emanuelgsouza** [comment]
I would like to help with Portuguese translation

**Comment by bazzadp** [comment]
> I would like to help with Portuguese translation

Good stuff @emanuelgsouza . Just sent you the invite and same instructions to you as I [said above](https://github.com/HTTPArchive/almanac.httparchive.org/issues/923#issuecomment-763150946).

**Comment by soulcorrosion** [comment]
Hi all, thanks for this amazing project. I would like to help in anyway I can. I'm portuguese, so I can lend a hand. How can I join the Web Almanac team?

**Comment by bazzadp** [comment]
> Hi all, thanks for this amazing project. I would like to help in anyway I can. I'm portuguese, so I can lend a hand. How can I join the Web Almanac team?

I've just sent you the invite to the Translators team there @soulcorrosion . The Portuguese translation is being tracked over at #505 so once you accept that invite, you should be able to edit the first comment in that issue to put your name beside a chapter to "claim it" for translation. Then follow the instructions and submit the PR with translated markdown!

Other than translations, 2020 edition is winding down, though there are still a few [open development issues](https://github.com/HTTPArchive/almanac.httparchive.org/issues?q=is%3Aissue+is%3Aopen+label%3Adevelopment) if interested in that.

And if interested in the 2021 edition then we've [started putting out early feelers for that](https://github.com/HTTPArchive/almanac.httparchive.org/issues/1954) and also have [an interest form](https://forms.gle/M4MxvGRvsEtBcBwr7) for when that ramps up more.

Let us know if you're any questions! We also have [a Slack channel](https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA) where you can reach out to other contributors, though again it's a little quite at he moment as we're between editions.

**Comment by soulcorrosion** [comment]
Thanks for the detailed explanation @bazzadp. I'll try to help with what I can.

**Comment by ucusita** [comment]
I can help for Spanish translations

**Comment by ademarCardoso** [comment]
I would like to help with Portuguese translation 🚀 🇧🇷

**Comment by bazzadp** [comment]
Welcome @ucusita and @ademarCardoso ! Thanks for volunteering. I just sent you invites to the team which will give you access to edit issues to claim a chapter.

The Spanish progress is being tracked in #498 
The Portuguese progress is being tracked in #505 

Jump on over to those issues, introduce yourself and grab a chapter to start translating.

**Comment by fatmabadri** [comment]
Hi @bazzadp , I would like to start with Turkish translation.

**Comment by bazzadp** [comment]
> Hi @bazzadp , I would like to start with Turkish translation.

Great stuff. I've sent you the invite to the team so you can be assigned issues and edit the issues. The Turkish translation is being tracked in #1044 but it's mot be started yet, so you'll need to do the templates first before you can start the chapter translations.

**Comment by fatmabadri** [comment]
Thank you @bazzadp. I will go through some documentation and create the template.

**Comment by carloscastromx** [comment]
I could help with the Spanish translations of both the 2019 and 2020 editions.

**Comment by bazzadp** [comment]
> I could help with the Spanish translations of both the 2019 and 2020 editions.

Excellent @carloscastromx - that’s being tracked over in #498 and I’ve sent you invites to the GitHub team. We’re already got all the basics set up for that so accept the invite, fork the repo, and grab a chapter from `src/content/en/` and translate and copy to `src/content/es/` and submit a PR.

Feel free to ask any questions in #498 !

**Comment by excellenseo** [comment]
Hi! I can help with translations in French and Spanish.

**Comment by MeFoDy** [comment]
Hi! I can help with Russian translations.

**Comment by bazzadp** [comment]
Good stuff @MeFoDy . That translation is being tracked over at #587 and I’ve just sent you an invite to the GitHub team so you can edit that first comment to put your name besides a chapter to translate.

**Comment by tunetheweb** [comment]
@HTTPArchive/translators the first 2021 chapter (PWA) is now ready for Translation if anyone wants to pick that up for their language. The Translation issues have been updated with the 2021 task (though not all chapters are ready yet).

**Comment by ksakae1216** [comment]
I can help for Japanese translations 🇯🇵

**Comment by tunetheweb** [comment]
We now have five 2021  chapters ready for translating:
- PWA
- Third Parties
- Structured Data
- Resource Hints
- Capabilities

Should have some more coming soon.

**Comment by kartashovio** [comment]
Hello! I can help with translation content to Russian.

**Comment by tunetheweb** [comment]
Excellent @kartashovio - just invited you to the team and head on over to #587 where we're tracking that and stick your name beside a 2021 chapter to claim it and away you go!

**Comment by jaideepghosh** [comment]
Hello @tunetheweb! I can help to translate content to Hindi.

**Comment by tunetheweb** [comment]
> jaideepghosh

Just sent you the GitHuh invite @jaideepghosh . The Hindi translation is being tracked in #1402 and the site was all set up for Hindi but none of the chapters have been translated yet - so pick your favourite one and go.

**Comment by nikolai-shabalin** [comment]
Hi! I can help with Russian translations for CSS 2022.

**Comment by tunetheweb** [comment]
Great! Head on over to #587 where Russian is being track. I've stuck your name beside the CSS chapter and sent you an invite to the team so you can edit issues too. More details here: https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide

**Comment by webmatter-it** [comment]
Hi, I would like to help translate into Italian (especially the chapter on fonts).

**Comment by tunetheweb** [comment]
Cool! Italian is being tracked at #585. I've stuck your name beside the Fonts chapter and sent you an invite to the team so you can edit issues too.

More details here: https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide but feel free to ask any questions!

**Comment by luckybai** [comment]
Hi! I can help with Simplified Chinese translations for WebAssembly 2022.

**Comment by tunetheweb** [comment]
Excellent. That;s being tracked over at #1067 and sent you an invite and put your name against that chapter.

**Comment by Levix** [comment]
Hi! @tunetheweb. I can help with Simplified Chinese translations for Part II Chapter 14 Security 2022. 
I need to submit a security chapter markdown document PR in the zh-CN folder for subsequent proofreading.
This is the PR:
https://github.com/HTTPArchive/almanac.httparchive.org/pull/3278

**Comment by decrecementofeliz** [comment]
Hello from Spain! I'd like to help with Spanish translation, specifically Privacy chapter to begin with. Thanks!

**Comment by tunetheweb** [comment]
> Hello from Spain! I'd like to help with Spanish translation, specifically Privacy chapter to begin with. Thanks!

Great stuff, we're tracking that over at #498 and tagged you in that, and sent you an invite to the repo so you can open branches here. More details here: https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide

**Comment by burakguneli** [comment]
Hello hello, I would like to help translate the latest Almanac into Turkish!

**Comment by neriiavr** [comment]
Hi, I would like to contribute as a Turkish translator!

**Comment by tunetheweb** [comment]
@burakguneli / @neriiavr I've sent you an invite to the Github org so you can edit issues and open branches on this repo.

The Turkish translation is being tracked at #1044 . The base templates have already been translated so stick your name against a chapter and start translating! More details in the [Translators' Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide).

Let me know if you've any questions!

**Comment by Ti0T3ch** [comment]
Hi, I want to contribute to the Portuguese translation!

**Comment by ozzmoman** [comment]
Hi, I want to contribute to the Persian translation!

**Comment by tunetheweb** [comment]
> Hi, I would like to contribute as a Persian translator!

Great! We don't have currently have any PErsian translations, we'd need to translate the base templates first before looking at any of the chapters:

> These are the core templates - without which we cannot release any translated chapters. They are in the [language specific templates directory](https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/templates):
>
> - base.html
> - base_chapter.html
> - error.html
>
> **2022**
> - base.html
> - contributors.html
> - index.html
> - table_of_contents.html

> **2021**
> - base.html -
> - contributors.html
> - index.html
> - table_of_contents.html
>
> **2020**
>- base.html
>- contributors.html 
>- index.html
>- table_of_contents.html
>
> **2019**
> - base.html
> - contributors.html
> - index.html
> - table_of_contents.html

This looks like a lot, but the year-specific ones are vary short.

After you open a PR for those, we can merge that, and look at the individual chapters. We'll also open a Persian-specific issue then.

**Comment by ozzmoman** [comment]
Please guide where should I start.
Thankful

**Comment by DanielEyza** [comment]
Olá, tenho interesse em entrar para o time portuguese. @rviscomi

**Comment by DanielEyza** [comment]
Hello, I found out about the project and would like to join the team for translation into Portuguese. @rviscomi

**Comment by HakaCode** [comment]
> Hello, I found out about the project and would like to join the team for translation into Portuguese. @rviscomi

@tunetheweb 
Please, could you

**Comment by tunetheweb** [comment]
Missed this. Sent you an invite to the project and the (Brazilian) Portuguese translation is being tracked at #505

**Comment by Alireza-Naji** [comment]
I can help for Persian translations. 2024 edition. if can guide me how can I do. thanks
https://almanac.httparchive.org/en/2024/structured-data

**Comment by tunetheweb** [comment]
Unfortunately Persian is not one of the languages supported by the Web Almanac yet. So before we can translate a chapter, we’d need to translate all the scaffolding to allow the site to run in that language.

That’s basically a matter of cloning any of the `src/templates` language files (e.g. `src/templates/en` for Persian) to a `src/templates/fa` directory for Persian and translating all the phrases in the blocks to Persian. You can look at the other languages to get an idea. It looks like a lot of files but most of them are just small phrases that we slot in at the appropriate places. You can also skip the Methodology and Accessibility Statement files for now.

Once that’s done the site can run in Persian and we can then translate individual chapters. This is a one off thing and will also make it much easier for future Persian speakers to contribute chapters.

I’m happy to help work through this with you from a technical perspective (I have no Persian myself!). So let me know if that appeals and we can open an issue for that and we can work through it.

**Comment by Alireza-Naji** [comment]
Thank you for your thoughtful reply. I’m very enthusiastic about contributing to the Persian translation. I’m prepared to take on the substantial work ahead and collaborate closely with your team. To get started, I’d like to begin with translating a few smaller files to familiarize myself with the site’s structure. It’s also worth mentioning that I have extensive expertise in SEO and am well-versed in professional concepts related to structured data and Accessibility. I look forward to receiving further guidance and appreciate your attention to multilingual support, especially for Persian.

https://www.linkedin.com/in/alirezanaji

**Comment by tunetheweb** [comment]
Excellent. I've sent you an invite to the repo and opened issue #3890. Let's continue the discussion in that issue.

**Comment by amineatia** [comment]
Hi team, I’m a Web Performance Consultant and UX Designer at Agence Web Performance in France, and I’d like to contribute as a translator for the French version this year. Please let me know how I can join and support the project.

**Comment by katopen0910** [comment]
Hello, I learned about the project and would like to join the team for translation into Russian and German

**Comment by tunetheweb** [comment]
Thank you @katopen0910 !

Russian is all set up and being tracked in #587
German was requested before, but never started, so there's a bit more to do for that one as none of the base templates have been translated yet. It's being tracked over at #1300

**Comment by braintrue** [comment]
Korean translation volunteer is here.

**Comment by tunetheweb** [comment]
Hi @braintrue Korean translation was being tracked in #926 . Some people have said they'd look but unfortunately no work ever reach the stage of a pull request. So this would need to be started from scratch with the base templates translated first, before any of the chapters can be translated.

**Comment by braintrue** [comment]
Thank you!
Then if i request translation in Korean about base template first(merge my branch into main), can I start to translate chapter then?

**Comment by tunetheweb** [comment]
Yes. There’s a few different base templates. Basically follow the order of translations in #926 and do all the “core templates” section in one PR.

**Comment by DataDish404** [comment]
Translator to Spanish or Italian here I go!
